compose/ui/ui-test/src/skikoTest/kotlin/androidx/compose/ui/test/IgnoreWebTest.kt

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2025 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.compose.ui.test
+
+@OptionalExpectation
+expect annotation class IgnoreWebTest()
```

Comment:
Nitpick: Let's use the same pattern between modules. Let's have separate annotations for each build target (IgnoreJs + IgnoreWasm) and just apply both where required.


But it's minor, so up to you "Web" in name is descriptive enough